Sources/MapboxMaps/Foundation/Camera/CameraAnimationDelegate.swift

```diff
@@ -6,14 +6,24 @@ public typealias AnimationCompletion = (UIViewAnimatingPosition) -> Void
 // MARK: CameraAnimatorDelegate Protocol
 internal protocol CameraAnimatorDelegate: class {
 
-    /**
-    This delegate function notifies that the completion block needs to be scheduled
+    /// The current camera of the map
+    var camera: CameraOptions { get }
 
-    - Parameter animator: The current animator that this delegate function is being called from
-    - Parameter completion: The completion block that needs to be scheduled
-    - Parameter animatingPosition  The position of the animation needed for the closure
-    */
-    func schedulePendingCompletion(forAnimator animator: CameraAnimator,
+    /// Ask the map to set camera to new camera
+    func jumpTo(camera: CameraOptions)
```

Comment:
I think we should consider splitting up the responsibilities of this protocol. Some of them could be fulfilled by injecting each animator with the MapboxMap. The rest (the ones that are UIView-related could be contained in a more focused protocol.

Reply:
I agree with this. Could we perhaps do it as part of subsequent PR?

Reply:
Ticketed : https://github.com/mapbox/mapbox-maps-ios-internal/issues/643